GameBananaPage.html

```diff
@@ -166,7 +166,7 @@
 	</li>
 	<li>
 		<b>Railed Move Block</b> -
-		Basically Move Block × Zip Mover. To clarify: this move block is meant to stay on a zip mover track, and only a player can move it by grabbing one of its sides and looking in certain directions. There are different steering modes, which will affect which side you need to grab / stand on to move the block. The horizontal mode will allow you to steer the block by grabbing the left or right side of it and looking left or right. The vertical mode will allow you to do the same by standing on the top side and looking up or crouching. The both mode turns on all three sides. (No you can't grab the bottom!)
+		Basically Move Block × Zip Mover. To clarify: this move block is meant to stay on a zip mover track, and only a player can move it by grabbing one of its sides and looking in certain directions. There are different steering modes, which will affect which side you need to grab / stand on to move the block. The horizontal mode will allow you to steer the block by grabbing the left or right side of it and looking left or right. The vertical mode will allow you to do the same by standing on the top side and looking up or crouching. The both mode turns on all three sides. With Xaphan Helper's Magnetic Ceilings, you can grab the bottom as well, and it acts like the top.
```

Comment:
i edited this file to remove the "(No you can't grab the bottom!)" that prompted this PR, but obviously i can't edit the actual gamebanana page